rpcserver.go

```diff
@@ -3961,6 +3967,20 @@ func (r *rpcServer) GetNetworkInfo(ctx context.Context,
 		return nil, err
 	}
 
+	// Sort the channels by capacity, and find the median.
```

Comment:
Maybe there should be a cap here in case the median is super high for some reason?

Reply:
You mean cap the number of channels we'll sort?

Reply:
Cap the median value amount

Reply:
This is just for the RPC call, why should we cap it?

Reply:
Question is whether to cap it for prefattach?